spec.html

```diff
@@ -1901,20 +1901,22 @@
       </emu-clause>
 
       <emu-clause id="sec-dataview.prototype.setbigint64">
-        <h1>DataView.prototype.setBigInt64 ( _byteOffset_, _value_ )</h1>
+        <h1>DataView.prototype.setBigInt64 ( _byteOffset_, _value_ [ , _littleEndian_ ] )</h1>
         <p>When the `setBigInt64` method is called with arguments _byteOffset_ and _value_, the following steps are taken:</p>
         <emu-alg>
           1. Let _v_ be the *this* value.
-          1. Return ? SetViewValue(_v_, _byteOffset_, *true*, `"Int64"`, _value_).
+          1. If _littleEndian_ is not present, let _littleEndian_ be *false*.
```

Comment:
Should this not be ` If _littleEndian_ is not present, let _littleEndian_ be *undefined*.` to match the other places in this file?

Reply:
yes, it's probably better if it's internally consistent (the existing DataView functions use both 'false' and 'undefined' as default values, and it shouldn't be an observable difference either way)